frontend/src/views/SqlEditor/TablePanel/TableView.vue

```diff
@@ -185,9 +185,14 @@ const handleExportBtnClick = (format: "csv" | "json") => {
   }
 
   const encodedUri = encodeURI(`data:text/${format};charset=utf-8,${rawText}`);
+  const rawDateISOString = new Date().toISOString();
+  const formatedDateISOString = rawDateISOString
+    .split(".")[0]
+    .replaceAll(":", "-");
+  const filename = `${currentTab.value.name}-${formatedDateISOString}`;
   const link = document.createElement("a");
 
```

Comment:
Consider adding an example in the comment? Can not imagine the result of the filename.